shared-services/provisionssmsessionmanager_policy_attachment.tf

```diff
@@ -1,9 +1,10 @@
 # ------------------------------------------------------------------------------
 # Attach to the ProvisionAccount role the IAM policy that allows
-# provisioning of the SSM Document resource required in this account.
+# provisioning of the SSM Document resource and set up SSM session
+# logging in this account.
```

Comment:
Just to align with the comment in `shared-services/provisionssmsessionmanager_policy.tf`. What drew my attention here was

```
# provisioning of the SSM Document resource and set up SSM session
```

so if you don't like this wholesale replacement I would at least like the `set` above changed.

```suggestion
# all of the permissions necessary to provision the SSM Document
# resource and set up SSM session logging in this account.
```

Reply:
Please see commit 1bd322c.